spec/tides-data-package.json

```diff
@@ -0,0 +1,2382 @@
+              "type": "string",
+              "examples": [
+                "{\n  \"title\": \"My Package Title\"\n}\n"
+              ]
+            },
+            "path": {
+              "title": "Path",
+              "description": "A fully qualified URL, or a POSIX file path.",
+              "type": "string",
+              "pattern": "^(?=^[^./~])(^((?!\\.{2}).)*$).*$",
+              "examples": [
+                "{\n  \"path\": \"file.csv\"\n}\n",
+                "{\n  \"path\": \"http://example.com/file.csv\"\n}\n"
+              ],
+              "context": "Implementations need to negotiate the type of path provided, and dereference the data accordingly."
+            },
+            "email": {
+              "title": "Email",
+              "description": "An email address.",
+              "type": "string",
+              "format": "email",
+              "examples": [
+                "{\n  \"email\": \"example@example.com\"\n}\n"
+              ]
+            },
+            "organization": {
+              "title": "Organization",
+              "description": "An organizational affiliation for this maintainer.",
+              "type": "string"
+            },
+            "role": {
+              "type": "string",
+              "default": "maintainer"
+            }
+          },
+          "required": [
+            "title"
+          ],
+          "context": "Use of this property does not imply that the person was the original creator of, or a contributor to, the data in the descriptor, but refers to the composition of the descriptor itself."
+        },
+        "examples": [
+          "{\n  \"maintainers\": [\n    {\n      \"title\": \"Joe Bloggs\"\n    }\n  ]\n}\n",
+          "{\n  \"maintainers\": [\n    {\n      \"title\": \"Joe Bloggs\",\n      \"email\": \"joe@example.com\",\n      \"role\": \"author\"\n    }\n  ]\n}\n"
+        ]
+      },
+      "licenses": {
+        "propertyOrder": 90,
+        "title": "Licenses",
+        "description": "The license(s) under which this package is published.",
+        "type": "array",
+        "minItems": 1,
+        "items": {
+          "title": "License",
+          "description": "A license for this descriptor.",
+          "type": "object",
+          "anyOf": [
+            {
+              "required": [
+                "name"
+              ]
+            },
+            {
+              "required": [
+                "path"
+              ]
+            }
+          ],
+          "properties": {
+            "name": {
+              "title": "Open Definition license identifier",
+              "description": "MUST be an Open Definition license identifier, see http://licenses.opendefinition.org/",
+              "type": "string",
+              "pattern": "^([-a-zA-Z0-9._])+$"
+            },
+            "path": {
+              "title": "Path",
+              "description": "A fully qualified URL, or a POSIX file path.",
```

Comment:
I have allowed the option of either, since some implementations down the road may involve a license kept privately at a relative filepath within a private repo (even if not the _ideal_ situation, a privately maintained license is certainly possible and not uncommon in govtech, so I don't want to box out users who may require supporting it). I updated the language here to adequately describe both cases.